src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -9,31 +10,18 @@
       ]
     }
   },
-  "dependencies": {
-    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160509"
-  },
   "tools": {
-    "dotnet-compile-fsc": {
-      "version": "1.0.0-preview2-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net45+win81",
-        "netstandard1.3"
-      ]
-    }
+    "dotnet-compile-fsc":"1.0.0-preview2-*"
```

Comment:
I think the next preview of the SDK is a while off from now, so we won't need to reference a new version for a while.  @piotrpMSFT @blackdwarf is this correct?